openpype/pipeline/frames.py

```diff
@@ -0,0 +1,83 @@
+"""Utility functions.
+
+This module contains utility functions dedicated to frame range operations
+"""
+import clique
+
+from openpype.lib import Logger
+
+log = Logger.get_logger(__name__)
+
+
+def get_frame_start_str(frame_start, frame_end):
+    """Get frame start string.
+
+    Args:
+        frame_start (int): first frame
+        frame_end (int): last frame
+
+    Returns:
+        str: frame start string
+    """
+    padding = len(str(frame_end))
+    return str(frame_start).zfill(padding)
+
+
+def get_asset_frame_range(asset_name=None, asset_id=None, fields=None):
```

Comment:
Pipeline functions should require what is minimum required to run properly and never use current context.
```suggestion
def get_asset_frame_range(project_name, asset_name, asset_doc=None):
```